docs/DafnyRef/UserGuide.md

```diff
@@ -144,6 +144,18 @@ Errors in earlier phases of processing typically hide later errors.
 For example, if a program has parsing errors, verification or compilation will
 not be attempted.
 
+### 25.6.1. The Dafny formatter
+
+Dafny supports a formatter, which for now only changes the indentation of every line in a Dafny file, so that it conforms
+to a specific model that matches the format of most of what the Dafny team wrote in the past.
```

Comment:
That the formatting style matches historical Dafny code is an interesting fact, but IMHO we should be more prescriptive here. If we intend that the formatter enforces "idiomatic" Dafny style, we should say something along the lines of:

```suggestion
Dafny supports a formatter, which for now only changes the indentation of every line in a Dafny file, so that it conforms
to the idiomatic Dafny code formatting style.
```

(or perhaps a bit weaker to indicate that the formatter only captures the indentation of the idiomatic Dafny code formatting style)

Reply:
> (or perhaps a bit weaker to indicate that the formatter only captures the indentation of the idiomatic Dafny code formatting style)

I like the way you phrased it, so I applied your changes.